bedrock/firefox/redirects.py

```diff
@@ -21,12 +21,9 @@ def firefox_channel(*args, **kwargs):
 
     redirect(r'^firefox/aurora/all/?$', 'firefox.all', to_kwargs={'channel': 'developer'}),
 
-    # bug 831810 & 1142583 & 1239960
-    redirect(r'^mwc/?$', 'firefox.os.devices', re_flags='i', query={
-        'utm_campaign': 'mwc-redirect',
-        'utm_medium': 'referral',
-        'utm_source': 'mozilla.org',
-    }),
+    # bug 831810 & 1142583 & 1239960, 1329931
+    redirect(r'^mwc/?$', 'https://support.mozilla.org/products/firefox-os'),
+    redirect(r'^MWC/?$', 'https://support.mozilla.org/products/firefox-os'),
```

Comment:
Instead of add another redirect for uppercase, you can just include `re_flags='i'`. It would also be nice to maintain the utm referral information I guess.